ci/nightly.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import smtplib
+from email.message import EmailMessage
+
+from tabulate import tabulate
+
+# Get parameters
+parser = argparse.ArgumentParser(description="Pushes nightly nvbug release")
+parser.add_argument("--cont", type=str, help="Merlin container")
+parser.add_argument("--qa", type=str, help="Container tests results")
+parser.add_argument("--server", type=str, help="Server to send email")
+parser.add_argument("--port1", type=str, help="Port to send email")
+parser.add_argument("--port2", type=str, help="Port to send email")
+parser.add_argument("--username", type=str, help="Username to send email")
+parser.add_argument("--password", type=str, help="Password to send email")
+parser.add_argument("--sender", type=str, help="Email sender")
+parser.add_argument("--receiver", type=str, help="Email receiver")
+args = parser.parse_args()
+
+# Set Table
+mydata = [
+    ["1", "NGC catalog URL of the container", args.cont],
+    ["2", "Version published to the NGC catalog", "nightly"],
+    ["3", "Expected publishing date and time", "COMPLETED"],
+    ["4", "QA results", args.qa],
+    ["5", "Security scan results", "??"],
+    ["6", "SWIPAT Approval", "??"],
```

Comment:
the following link is the nvbug that gives us SWIPAT approval, so you can add it in: https://nvbugswb.nvidia.com/NvBugs5/SWBug.aspx?bugid=3267056&cmtNo=